packages/generator-default/rollup.config.js

```diff
@@ -1,7 +1,6 @@
 /* @flow */
 
 import babel from 'rollup-plugin-babel'
-import uglify from 'rollup-plugin-uglify'
```

Comment:
I would be fine with removing this for development NODE_ENV but removing it entirely isn't what I want, it'll make the bundle output sizes really big